presentation/src/main/java/com/bunbeauty/presentation/viewmodel/base/DataState.kt

```diff
@@ -0,0 +1,3 @@
+package com.bunbeauty.presentation.viewmodel.base
+
+interface DataState
```

Comment:
can we place all in one file like in client pls 
Event
DataState
ViewDataState
ViewState

Reply:
fixed ✅